controllers/events.js

```diff
@@ -78,8 +79,14 @@ router.get('/event/', function (req, res) {
                     // If a logged in user requests this, see if they have liked this event
                     if (data.ok) {
                         response = data;
-                        return models.User.findOne({ where: { spotifyID: req.session.userID } });
+                        // Build google search query (with extra terms "crowd" & "stage" to get better images)
+                        var query = googleImageSearch.buildImageQuery(response.event.eventname + " crowd stage");
+                        // Make google search
+                        return googleImageSearch.makeRequest(query); 
```

Comment:
maybe catch the image response here and return some default image, if the api key runs out for the day